checker/src/main/java/org/checkerframework/checker/mustcall/JdkCompiler.astub

```diff
@@ -0,0 +1,6 @@
+import org.checkerframework.checker.mustcall.qual.*;
+
+package com.sun.tools.javac.processing;
+
+@MustCall({})
+public class JavacProcessingEnvironment implements ProcessingEnvironment, Closeable {}
```

Comment:
If we are adding this stub file, why do we need the `-AskipUses` on `ProcessingEnvironment` in `build.gradle`?